app/src/test/java/protect/card_locker/LoyaltyCardTest.java

```diff
@@ -0,0 +1,96 @@
+package protect.card_locker;
+
+import android.os.Parcel;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.math.BigDecimal;
+import java.util.Currency;
+import java.util.Date;
+
+import com.google.zxing.BarcodeFormat;
+
+import static org.junit.Assert.*;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(manifest=Config.NONE)
```

Comment:
Just curious, why this line? No other tests seem to use it.

Reply:
Wondering the same. It also seems to be [deprecated](https://robolectric.org/javadoc/latest/org/robolectric/annotation/Config.html#manifest()).

Reply:
Thank you for bringing this to my attention.

The @Config(manifest=Config.NONE) line was included to ensure that Robolectric tests run without relying on an Android manifest file. However, I see that the manifest field within the @Config annotation is marked as deprecated. This means that we should migrate to the preferred way of configuring builds as recommended in the Robolectric documentation.

I will update the tests to remove the deprecated usage and follow the new guidelines